### PR TITLE
Tweaks to Changelog

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -37,9 +35,10 @@ Added significant improvements for large organizations running on a single serve
 #### Favorite Channels
 - Added the ability to select Favorite Channels that appear at the top of the channels sidebar.
 
-#### Video and Audio Calling (Beta)
+#### Video and Audio Calling (early-preview)
 - Added early preview of video and audio calling option using self-hosted proxy.
 - Intended as working prototype for community development, not recommended for production.
+- Early preview does not include logging or detailed documentation.
```
Comment (jasonblais) [inline source/administration/changelog.md]: Is this line necessary? There's going to be docs letting people know how to set up WebRTC (the PR is already merged against 3.5 branch).

File: source/administration/changelog.md [hunk 2]
```diff
@@ -37,9 +35,10 @@ Added significant improvements for large organizations running on a single serve
 #### Favorite Channels
 - Added the ability to select Favorite Channels that appear at the top of the channels sidebar.
 
-#### Video and Audio Calling (Beta)
+#### Video and Audio Calling (early-preview)
```
Comment (jasonblais) [inline source/administration/changelog.md]: In my mind "Beta" is fine. We don't use "early-preview" anywhere else. But I'm only 1/5

File: source/administration/changelog.md [hunk 3]
```diff
@@ -51,13 +50,13 @@ Added significant improvements for large organizations running on a single serve
 ### Improvements
 
 #### iOS Apps
-- Channel and account settings, and channel header are now full screen modals for better visibility
+- Channel settings, account settings, and channel header render are now full screen modals for better visibility
```
Comment (jasonblais) [inline source/administration/changelog.md]: Sentence seems weird, did you mean `Channel settings, account settings, and channel header now render full screen for better visibility`

File: source/administration/changelog.md [hunk 4]
```diff
@@ -51,13 +50,13 @@ Added significant improvements for large organizations running on a single serve
...
 - [...] menu options now displayed larger for better usability
 - Keyboard doesn't automatically close when sending a message, letting you quickly send several messages in succession
 - When the "Download" link is clicked on files, a "Back" button lets users get back to the app
 
 #### Android Apps
-- Channel and account settings, and channel header are now full screen modals for better visibility
+- Channel settings, account settings, and channel header render are now full screen modals for better visibility
```
Comment (jasonblais) [inline source/administration/changelog.md]: Same here

**Comment by esethna** [comment]
+1

**Pull request merged**